Project/backend/config/llm/gemini.py

```diff
@@ -6,15 +6,7 @@
 
 
 def configure_genai():
-    """
```

Comment:
Did you intentionally remove the comments from the function, or was the branch not yet updated? @yashbhesaniya